<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinType</class>
 <widget class="QWidget" name="WinType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QStackedWidget" name="Other">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="WinConfOther">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0" colspan="2">
        <widget class="QTableWidget" name="TabWire">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>2</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>4</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>5</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>6</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>7</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>8</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="TextColor">
         <property name="text">
          <string>线夹颜色</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="TextNG">
         <property name="text">
          <string>遇不合格测试</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="labelType">
         <property name="pixmap">
          <pixmap resource="image.qrc">:/source/M1S0.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="TextOther">
         <property name="text">
          <string>其他参数</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextType">
         <property name="text">
          <string>电机类型选择</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BoxType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>M1S0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ABC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ABC-N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ABC-O</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ABC-Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S1L1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S1L2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1S1L3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="BoxTestNG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>暂停询问</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>继续测试</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WinConfProj">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="TextItemSelect">
         <property name="text">
          <string>项目选择与设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemDelete">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:yellow;font:Bold 15pt Ubuntu</string>
         </property>
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemSet">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:yellow;font:Bold 15pt Ubuntu</string>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemDcr">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>电阻</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemMag">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>反嵌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemInr">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>绝缘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemAcw">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>交耐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemDcw">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>直耐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemImp">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>匝间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemInd">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>电感</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemPwr">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemLvs">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>低启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnItemLck">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>堵转</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="Type">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="WinConfList">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0" columnminimumwidth="1,1,1,1,1">
       <item row="4" column="4">
        <widget class="QPushButton" name="BtnConfExit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>保存调入</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>型号管理</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="BtnInsert">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>添加型号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="4">
        <widget class="QTableWidget" name="TabTest">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>44</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>测试项目</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QTableWidget" name="TabFile">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>44</number>
         </attribute>
         <column>
          <property name="text">
           <string>电机名称</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>文件名称</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="BtnDelete">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>删除型号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>测试项目</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="TextCurrentType">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>当前型号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QLabel" name="TextTypeShow">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:yellow;font:Bold 15pt Ubuntu</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="BtnCheck">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3">
        <widget class="QLineEdit" name="EditTypeName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WinConfColor">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="TextColorSelect">
         <property name="text">
          <string>线夹颜色选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="TabColor">
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>新建列</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
